pkgs/development/python-modules/pypika/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, parameterized
+, unittestCheckHook
+}:
+buildPythonPackage rec {
+  pname = "pypika";
+  version = "0.48.9";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "kayak";
+    repo = "pypika";
+    rev = "v${version}";
+    hash = "sha256-9HKT1xRu23F5ptiKhIgIR8srLIcpDzpowBNuYOhqMU0=";
+  };
+
+  pythonImportsCheck = ["pypika"];
+
+  nativeCheckInputs = [
+    parameterized
+    unittestCheckHook
+  ];
+
+  meta = with lib; {
+    description = "PyPika is a python SQL query builder that exposes the full richness of the SQL language using a syntax that reflects the resulting query. PyPika excels at all sorts of SQL queries but is especially useful for data analysis";
```

```suggestion
    description = "A python SQL query builder";
```
It may be a bit long for description.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description

---

Oh yeah. They had fit a lot in that tiny repo description bix. :-)